lib/foreman_templates/engine.rb

```diff
@@ -16,6 +16,17 @@ class Engine < ::Rails::Engine
     initializer 'foreman_templates.register_plugin', :before => :finisher_hook do
       Foreman::Plugin.register :foreman_templates do
         requires_foreman '>= 1.12'
+
+        security_block :templates do
+          permission :import_templates, {
+            :template => [:import],
+            :"api/v2/template" => [:import]
+          }, :resource_type => 'Template' 
+        end
+
+        role = Role.where(:name => 'Manager').first
+        role.add_permissions!('import_templates') unless role.permissions.find_by_name('import_templates')
```

Comment:
@ares Hows the best way to do this? Looks like its fails because the import_template permissions isn't registered yet. I wanted to avoid a db migration but it looks like the only way to me..